modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_panel_group.yml

```diff
@@ -0,0 +1,45 @@
+id: az_paragraph_panel_group
+label: AZ Paragraph Panel Group
+migration_group: az_migration
+migration_tags:
+  - Quickstart Content Migration
+status: true
+
+source:
+  plugin: az_paragraphs_item
+  bundle: uaqs_panel_group
+  field_collection_names: 'field_uaqs_panel'
+
+destination:
+  plugin: 'entity_reference_revisions:paragraph'
+  default_bundle: az_accordion
+
+process:
+  field_az_accordion:
+    -
+      plugin: sub_process
+      source: field_uaqs_short_title
+      process:
+        delta: delta
+        title: value
+        body_format:  
```

Comment:
Should the `body_format` settings be on the `field_uaqs_summary` rather than `field_uaqs_short_title`?

Reply:
`body_format` settings under `field_az_accordion`, that is why added it under sub_process process plugin. Other two are custom plugins. Migration will assign default `az_standard` to `field_az_accordion`.

Reply:
This still needs updating

Reply:
Done